projects/js-packages/connection/composer.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "automattic/js-connection",
+	"description": "Jetpack Connection Component.",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "1.1.x-dev"
+	},
+	"scripts": {
+		"test-js": [
+			"Composer\\Config::disableProcessTimeout",
+			"yarn install",
+			"yarn test"
+		],
+		"test-coverage": [
+			"Composer\\Config::disableProcessTimeout",
+			"yarn install",
+			"yarn nyc --reporter=clover -x './!(node_modules)/**/test/*.jsx' --report-dir=\"$COVERAGE_DIR\" yarn test"
```

Comment:
Looks like there's something wrong with the coverage, it's producing an empty coverage file (which then confuses CodeClimate).